content/pt-br/docs/concepts/policy/pid-limiting.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Limites e reservas de ID de processo
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.20" state="stable" >}}
+
+O Kubernetes permite limitar o número de IDs de processo (PIDs) que um
+{{< glossary_tooltip term_id="Pod" text="Pod" >}} pode usar.
+Você também pode reservar um número de PIDs alocáveis para cada {{< glossary_tooltip term_id="node" text="node" >}}
```

Comment:
No problema translate this tag text="node"

Reply:
Done

Reply:
Hi @DaviMatheus Please, revert "texto". When I wrote tag I meant the content of the tag "node" and not the text "text=". Please, check the text below. Sorry about that.